src/types/markdown.css

```diff
@@ -0,0 +1,9 @@
+@value form_css: "../form.css";
+
+.input {
+    composes: input from form_css;
```

Comment:
Please add a max-height of some reasonable size. The textarea component would probably be a good example for that.